### PR TITLE
288 error when labels dont go from 0 to l 1

### PR DESCRIPTION
**Pull request by vincentblot28**

### CODE REVIEW

File: mapie/classification.py [hunk 1]
```diff
@@ -565,7 +565,7 @@ def _fit_and_predict_oof_model(
 
     def _get_true_label_cumsum_proba(
         self,
-        y: NDArray,
+        y: ArrayLike,
```
Comment (thibaultcordier) [inline mapie/classification.py:568]: Is this change justified?

Comment (vincentblot28) [reply mapie/classification.py:568]: yes, otherwise the typing check failes

Comment (vincentblot28) [reply mapie/classification.py:568]: as y is directly the output of the model it may be more complexe than just an array

**Review by thibaultcordier — APPROVED** [review]
Approve the bug fix.

**Pull request merged**